### ISSUE TITLE
Change password shows status message in 

### ISSUE THREAD

**Issue by jn3va** [Geektoolkit/DynaframeProIssues#105]
When you change the system password from the Settings page, after clicking the check mark button the following message is presented:

 Произошла неизвестная ошибка.

Google translate indicates this messages means "An unknown error has occurred" but the password seems to have been changed.

Other functions seem to show the correct language for status.  e.g. change slide duration shows:  "SlideshowTransitionTime was successfully changed to 40000"

**Comment by Geektoolkit** [comment]
Yeah this is due to the notification system we used in the current webUI. This will go away with the new webUI thankfully. Thankyou for reporting it, I'll keep this open till our next big release which deprecates that UI and replaces it with a much better one

**Comment by Geektoolkit** [comment]
This is fixed in the alpha, we no longer use the library that caused this
